lib/gen_stage.ex

```diff
@@ -1196,10 +1178,12 @@ defmodule GenStage do
 
   ## Options
 
-    * `:cancel` - `:permanent` (default) or `:temporary`. When `:permanent`,
-      the consumer exits when the producer cancels or exits. In case
-      of exits, the same reason is used to exit the consumer. In case of
-      cancellations, the reason is wrapped in a `:cancel` tuple.
+    * `:cancel` - `:permanent` (default), `:transient` or `:temporary`.
```

Comment:
Being way nitpicky here but we could have a Oxford comma since you're using it in other places :P